torch/_dynamo/variables/user_defined.py

```diff
@@ -1340,6 +1340,34 @@ def __init__(self, value, fields=None, **kwargs) -> None:
             fields = {}
         self.fields = fields
 
+    def as_python_constant(self):
```

Comment:
Just a question: when we're handling register_constant, are we going to do some full blown dynamo bytecode reconstruction? Because I'm thinking we might end up there in the future (doesn't have to happen in this PR)

Reply:
This was discussed during meeting -- for pre-existing objects this is easy, we just reuse the objects. Otherwise yeah we might have to bite the bullet and implement full-blown bytecode-based reconstruction or sth.